src/User/resources/i18n/ru/usuario.php

```diff
@@ -185,9 +185,9 @@
     'Your account has been connected' => 'Ваш аккаунт присоединён',
     'Your account has been created' => 'Ваш аккаунт создан',
     'Your account has been created and a message with further instructions has been sent to your email' => 'Аккаунт создан и дальнейшие инструкции высланы на вашу электронную почту',
-    'Your account on {0} has been created' => '',
-    'Your confirmation token is invalid or expired' => 'Токен подтверждения невалиден или истёк срок его действия',
-    '{0, date, MMMM dd, YYYY HH:mm}' => '',
+    'Your account on {0} has been created' => 'Ваш аккаунт на сайте "{0}" был успешно создан',
+    'Your confirmation token is invalid or expired' => 'Ваша ссылка устарела или является ошибочной',
+    '{0, date, MMMM dd, YYYY HH:mm}' => '{0, date, MMMM dd, YYYY HH:mm}',
     'Class "{0}" does not exist' => '@@@@',
     'Registration ip' => '@@@@',
     'Rule class can not be instantiated' => '@@@@',
```

Comment:
[Is it used in the source code?](https://github.com/yiisoft/yii2/blob/master/framework/console/controllers/MessageController.php#L84)